Lumina/Lumina/UI/Extensions/InterfaceHandlerExtension.swift

```diff
@@ -19,6 +19,7 @@ extension LuminaViewController {
     }
 
     @objc func handleTapGestureRecognizer(recognizer: UITapGestureRecognizer) {
+        self.delegate?.tapped(controller: self)
```

Comment:
I might be wrong here, and it likely needs to also change in the rest of the function, but can we get rid of `self.` here? We're not inside the scope of a closure, but maybe the `@objc` forces it?